mage_ai/io/chroma.py

```diff
@@ -0,0 +1,127 @@
+
+import logging
+from typing import Dict
+
+import chromadb
+from pandas import DataFrame
+
+from mage_ai.io.base import BaseIO
+from mage_ai.io.config import BaseConfigLoader, ConfigKey
+
+logger = logging.getLogger(__name__)
+
+
+class Chroma(BaseIO):
+    def __init__(
+            self,
+            collection: str,
+            path: str = None,
+            verbose: bool = True,
+            **kwargs,) -> None:
+        """
+        Initializes settings to connect to chroma db.
+        """
+        super().__init__(verbose=verbose)
+        self.collection = collection
+        if path is None:
+            self.client = chromadb.Client()
+        else:
+            self.client = chromadb.PersistentClient(path=path)
+
+    @classmethod
+    def with_config(cls, config: BaseConfigLoader) -> 'Chroma':
+        return cls(
+            collection=config[ConfigKey.CHROMA_COLLECTION],
+            path=config[ConfigKey.CHROMA_PATH],
+        )
+
+    def __convert_to_multiple_rows(self, data: Dict, column_name: str, item_length: int):
```

Comment:
Could you show an example structure of data in the comments?
Is data[column_name] a list of lists?

Reply:
Added an example for format of `data` read from chroma